xml/hwbp_registers.xml

```diff
@@ -1132,18 +1132,20 @@
     <register name="Interrupt Trigger" short="itrigger" address="0x7a1">
         This register is accessible as \RcsrTdataOne when \FcsrTdataOneType is 4.
 
-        This trigger may fire on any of the interrupts configurable in \Rmie
-        (described in the Privileged Spec) or the NMI. The interrupts to fire on are
-        configured by setting the same bit in \RcsrTdataTwo as would be set in
-        \Rmie to enable the interrupt.
+        This trigger may fire on any of the interrupts described in the
+        Privileged Spec, or the NMI. The interrupts to fire on are configured by
+        setting the bit in \RcsrTdataTwo whose index would be written to the
+        Exception Code field of the mode-relevant {\tt cause} register when the
+        trap is taken. For interrupts whose numbers are translated, that means
+        the number checked is the one in the mode that the trap handler executes
+        in.
 
         Hardware may only support a subset of interrupts for this trigger.  A
         debugger must read back \RcsrTdataTwo after writing it to confirm the
         requested functionality is actually supported.
 
         The trigger only fires if the hart takes a trap because of the
-        interrupt. (E.g.\  it does not fire when a timer interrupt occurs but that
-        interrupt is not enabled in \Rmie.)
+        interrupt. (E.g.\ it does not fire for an interrupt that is masked.)
```

Comment:
If possible, I like to avoid "masked" because sometimes you have positive masks `(A & mask)` and sometimes you have negative masks `(A & ~mask)` so it can be ambiguous.  What you have is probably not that ambiguous but how about using something closer to the original language?  That also applies to global enable (mstatus.MIE) rather than just local enables (mie).
```suggestion
        interrupt. (E.g.\ it does not fire for an interrupt that is not enabled.)
```

Reply:
disabled? Done.